README.md

```diff
@@ -2,9 +2,22 @@
 
 Gem to help with A/B testing on the GOV.UK platform.
 
-## Technical documentation
-
-### Installation
+## Contents
+
+* [Installation](#installation)
+* [Pre-requisites](#pre-requisites)
+* [Usage](#usage)
+  * [Outline](#outline)
+  * [Example A/B test logic](#example-ab-test-logic)
+* [Acceptance testing](#acceptance-testing)
+    * [Test helpers](#test-helpers)
+      * [RSpec](#rspec)
+      * [Minitest](#minitest)
+* [Running the test suite](#running-the-test-suite)
+* [Checking your A/B test in a browser](#checking-your-ab-test-in-a-browser)
+* [API documentation](#api-documentation)
```

Comment:
GitHub does a table of contents automatically, so it might not be needed here.

Example for this repo: 
![image](https://user-images.githubusercontent.com/1732331/130585966-487e408e-fddc-4ae6-a2d1-454c9449c050.png)

Reply:
It also gets rendered in the Dev Docs but I cba to maintain a TOC.